packages/api/src/controllers/customAnimalBreedController.js

```diff
@@ -44,6 +47,78 @@ const customAnimalBreedController = {
       }
     };
   },
+
+  addCustomAnimalBreed() {
+    return async (req, res) => {
+      const trx = await transaction.start(Model.knex());
+      try {
+        const { farm_id } = req.headers;
+        let { breed } = req.body;
+        const { default_type_id, custom_type_id } = req.body;
+        breed = baseController.checkAndTrimString(breed);
+
+        if (!breed) {
+          await trx.rollback();
+          return res.status(400).send('Animal breed must be provided');
+        }
+        if (default_type_id && custom_type_id) {
+          await trx.rollback();
+          return res.status(400).send('Only default_type_id or custom_type_id should be specified');
+        }
+        if (!default_type_id && !custom_type_id) {
+          await trx.rollback();
+          return res
+            .status(400)
+            .send('One of default_type_id or custom_type_id should be specified');
+        }
+
+        let breedDetails;
+        if (custom_type_id) {
+          const customType = await CustomAnimalTypeModel.query().findById(custom_type_id);
+          if (customType?.farm_id === farm_id) {
+            breedDetails = { custom_type_id, breed };
+          } else {
+            await trx.rollback();
+            return res
+              .status(403)
+              .send('User does not have permission to access to requested resource');
+          }
```

Comment:
I know I'm being super picky, but can we make it consistent? 😅 

https://github.com/LiteFarmOrg/LiteFarm/blob/18923333ec3b81cc95fa429f5501ffb3ad1dcc72/packages/api/src/controllers/animalController.js#L90-L95

Reply:
Thanks! 😓 .. should be more consistent now